specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySingleResource.json

```diff
@@ -93,7 +93,10 @@
               "enableChatWithUs": false,
               "priority": 2,
               "lastUpdateTime": "2018-11-08T00:00:00Z",
-              "hirStage": "active"
+              "hirStage": "active",
```

Comment:
We don't need to show these new fields in this API example. The API flow is different for SingleResource events.